apps/drupal/patternlab.theme

```diff
@@ -4,3 +4,45 @@
  * @file
  * Functions to support theming in the Pattern Lab theme.
  */
+
+/**
+ * Implements hook_theme_suggestions_HOOK_alter().
+ */
+function patternlab_suggestions_block_alter(array &$suggestions, array $variables) {
+  // Provide suggestion for block templates by custom block type.
+  if (!empty($variables['elements']['content']['#block_content'])) {
+    $block = $variables['elements']['content']['#block_content'];
+    // Add `block--BLOCK-TYPE.html.twig`
```

Comment:
We're super sure that Drupal 8 doesn't do this already?

Reply:
Triple double sure!